client/components/forms/form-phone-input/index.jsx

```diff
@@ -17,6 +17,7 @@ export class FormPhoneInput extends Component {
 		initialCountryCode: PropTypes.string,
 		initialPhoneNumber: PropTypes.string,
 		countriesList: PropTypes.array.isRequired,
+		isHideLabels: PropTypes.bool,
```

Comment:
Would this make sense as simply 'hideLabels'?

Reply:
On second thought, I removed this prop altogether so that we can keep using the existing labels.